src/type.ts

```diff
@@ -0,0 +1,56 @@
+export interface Product {
+    id: number;
+    title: string;
+    description: string;
+    price: number;
+    images: [];
+    creationAt: string;
+    updatedAt: string;
+    category: {};
```

Comment:
You need to define type of your category.
export interface Product {
// ...
category: Category
}

![image](https://github.com/DavidTereshchenko/fe-react-2024/assets/108006836/b99531bf-cf52-4c82-96bb-3e67de3e027f)

Reply:
fixed